language-design.md

```diff
@@ -59,7 +135,17 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
-
+The post on grayscale tells us that the community contributes to the process
+of API design. In the post, users were able to vote on which naming to use
+for grayscale. It also shows that people in the community have varied (and
+sometimes strong) opinions on things as simple as naming a function for 
+grayscale. We also were rather surprised that the rgb function won by so many
+votes since we both thought that black was by far the clearest naming choice.
+This reveals that different communities have different shared ideas that make
+some choices clearer to members of the community but not to outsiders.  We
```

Comment:
This is pretty insightful. A word might mean very different things from one domain to the next, and while it might be somewhat hard to understand or even illogical, if it has an established meaning in a domain then changing it will require a rather compelling reason.